java/daikon/LogHelper.java

```diff
@@ -22,6 +22,7 @@ private LogHelper() {
 
   // Class variables so user doesn't have to use "Level." prefix.
```

Comment:
Who is the user?  The programmer editing this file, or a user of Daikon, or someone else?

Reply:
code was deleted as part of previous comment resolution